fuel-core-interfaces/src/lib.rs

```diff
@@ -10,18 +10,8 @@ pub mod sync;
 pub mod txpool;
 
 pub mod common {
-    #[doc(no_inline)]
-    pub use fuel_asm;
-    #[doc(no_inline)]
-    pub use fuel_crypto;
-    #[doc(no_inline)]
-    pub use fuel_merkle;
-    #[doc(no_inline)]
-    pub use fuel_storage;
-    #[doc(no_inline)]
-    pub use fuel_tx;
-    #[doc(no_inline)]
-    pub use fuel_types;
     #[doc(no_inline)]
     pub use fuel_vm;
+    #[doc(no_inline)]
+    pub use fuel_vm::*;
```

Comment:
do we really want everything from vm included here? This will include a lot more stuff into `common` than before.

Reply:
Yeah, the question here do we want to not worry about potential future re-exports. Or we want to specify each export manually and update it manually?

Reply:
Ideally we'd have a standard approach to bundling fuel-* usages (e.g. something like `common`), that way they are easier to re-export downstream. But this doesn't really hurt that much right now.